### ISSUE TITLE
VSCode Terminal Not Functioning & Extensions All Broken / Not Showing Up [Total Breakage]

### ISSUE THREAD

**Issue by techwithtim** [microsoft/vscode#147459]
After opening VSCode today my terminal and all extension features are completely broken. I cannot view or search for extensions, the pane is constantly loading. I also cannot create/delete or use my VSCode terminal. 

VSCode was functioning fine yesterday. I have made no major changes to my setup/environment.

I have attached a video that depicts exactly what occurs.

https://user-images.githubusercontent.com/35348871/163435252-344943f2-24ac-4efb-b0fb-2d2b9fa4e6d5.mp4

Does this issue occur when all extensions are disabled?: Yes

Logs:
[extHostLog.txt](https://github.com/microsoft/vscode/files/8490907/extHostLog.txt)
[mainLog.txt](https://github.com/microsoft/vscode/files/8490908/mainLog.txt)
[rendererLog.txt](https://github.com/microsoft/vscode/files/8490909/rendererLog.txt)
[sharedLog.txt](https://github.com/microsoft/vscode/files/8490910/sharedLog.txt)

- VS Code Version: 1.66.2
- OS Version: Windows_NT x64 10.0.19043

Steps to Reproduce:

1. Open VSCode

**Comment by rebornix** [comment]
@techwithtim thanks for the video, it's really helpful for troubleshooting.

Starting from @meganrogge since the terminal can not be created.

**Comment by techwithtim** [comment]
Great, let me know if you need anything else. I attached the logs in the issue comment

**Comment by techwithtim** [comment]
Update: I was able to fix this issue by changing the following key in my registry: **Computer\HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Services\Null**

I had previously changed the value of this to be 4, to fix a memory leak in windows 10. Turns out that after restarting my computer with this new value my git command completely broke and as seen in the video it broke vscode features as well. 

To fix this I changed the value to 1, restarted and now everything is functioning as normal.
